### PR TITLE
Add basic layout

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
   },
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "node ./node_modules/.bin/design build icons && next build",
```
[the remark this answers is not in the document]
Comment (Joozty) [reply package.json:59]: Can you please elaborate how do you imagine it should work? Ideally, put here code change suggestion.

File: modules/layout/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  --app-bar-skip: 3.5rem;
+
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+  padding-top: var(--app-bar-skip);
+  background: var(--gray-200);
+}
+
+.main {
```
[the remark this answers is not in the document]
Comment (Joozty) [reply modules/layout/styles.module.css:12]: Container wraps the whole screen and makes overflow on x axis happen.  We couldn't delegate this work to the parent element (`<div class="_next" />`). However, with this approach, I would say we are less dependent on next. 🤔

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply package.json:59]: I think in the dependent projects commands are executable by the binary name:

```suggestion
    "build": "@oacore/design build icons && next build",
```

[the remark this answers is not in the document]
Comment (Joozty) [reply package.json:59]: nope this doesn't work

**Comment by Joozty** [comment]
I restructured it little bit. 

To answer your questions:
>why the card (form container) is flex?
it was easier for me to style

> how it's centred if the main container is not flex?
it is flex now and centering is done via auto margins

> I noticed the card is shifted to the bottom a bit on my screen if the window is not full height. See the screenshot attached.
thanks it should be better now.

**Pull request merged**